<launch>
  <arg name="can_tx_topic" default="/transmit/frame"/>
  <arg name="operation_mode_topic" default="/system/operation_mode/state"/>
  <arg name="timer_period_ms" default="100"/>


  <node pkg="dtg_can_sender" exec="dtg_can_sender_node" name="dtg_can_sender"
        output="screen" respawn="true">
    <param name="can_tx_topic" value="$(var can_tx_topic)"/>
    <param name="operation_mode_topic" value="$(var operation_mode_topic)"/>
    <param name="timer_period_ms" value="$(var timer_period_ms)"/>
    <!-- 실제 CAN 드라이버 사용 시 -->
    <!-- <remap from="/transmit/frame" to="/can0/transmit/frame"/> -->

  </node>
</launch>
